internal/backend/db/dbgorm/projects.go

```diff
@@ -110,7 +110,10 @@ func (gdb *gormdb) updateProject(ctx context.Context, p *scheme.Project) error {
 		if err := tx.isCtxUserEntity(ctx, p.ProjectID); err != nil {
 			return err
 		}
-		return tx.db.Updates(p).Error
+
+		data := map[string]any{"Name": p.Name}
+		allowedFields := []string{"Name"}
+		return tx.db.Model(&scheme.Project{ProjectID: p.ProjectID}).Select(allowedFields).Updates(data).Error
```

Comment:
Could you explain what are you trying to do here?
Why not just regular update(column,value) or update(object) ?